pipeline.py

```diff
@@ -72,31 +94,115 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+
+        for article in articles:
+            processed_tokens = self._process(article.get_raw_text())
+
+            tokens_cleaned = []
+            tokens_single_tagged = []
+            tokens_multi_tagged = []
+
+            for processed_token in processed_tokens:
+                tokens_cleaned.append(processed_token.get_cleaned())
+                tokens_single_tagged.append(processed_token.get_single_tagged())
+                tokens_multi_tagged.append(processed_token.get_multiple_tagged())
+
+            article.save_as(' '.join(tokens_cleaned), 'cleaned')
+            article.save_as(' '.join(tokens_single_tagged), 'single_tagged')
+            article.save_as(' '.join(tokens_multi_tagged), 'multiple_tagged')
```

Comment:
as it is stated in the 5.3 section of pipeline documentation, you are required to pass `ArtifactType` attributes as a `kind` argument to the `save_as() method`, this way code is more secure as it is protected from almost untraceable errors caused by simple typos

Reply:
Thanks, I fixed that